src/pages/List.js

```diff
@@ -9,12 +9,42 @@ import dayjs from 'dayjs';
 import './List.css';
 
 const List = props => {
-  const { shoppingList, setShoppingList, addDatePurchased } = useContext(
-    ListContext,
-  );
+  const {
+    shoppingList,
+    setShoppingList,
+    addDatePurchased,
+    isDuplicate,
+  } = useContext(ListContext);
   const { token } = useListToken;
   const today = dayjs();
 
+  const [filteredInput, setFilteredInput] = useState('');
+  const [existingItem, setExistingItem] = useState(false);
+
+  //1. useState to filter input
+
+  /*
+
+  2. Filter Function
+
+  const filterAr = ar => {
+  return ar.filter(
+    item =>
+      item.id.slice(0, filteredInput.length) === filteredInput,
+  );
+};
+
+*/
+  function handleFilterChange(event) {
```

Comment:
@prophen i'm sure we're still working on this but it wasn't working on my end.

Reply:
It was in the other branch. I just merged so we can work on one branch again.